ert_data/obs_loader.py

```diff
@@ -0,0 +1,95 @@
+import pandas as pd
+
+
+def data_loader_factory(observation_type):
+    """
+    Currently, the methods returned by this factory differ. They should not.
+    TODO: Remove discrepancies between returned methods.
+        See https://github.com/equinor/libres/issues/808
+    """
+    if observation_type == "GEN_OBS":
+        return load_general_data
+    elif observation_type == "SUMMARY_OBS":
+        return load_summary_data
+    elif observation_type == "BLOCK_OBS":
+        return load_block_data
+    else:
+        raise TypeError("Unknown observation type: {}".format(observation_type))
+
+
+def load_general_data(facade, observation_key, case_name):
+    observations = facade.get_observations()
+    data = pd.DataFrame()
+    if observation_key not in observations:
+        return data
+
+    obs_vector = facade.get_observations()[observation_key]
```

Comment:
`facade.get_observations()` should become now `observations`

Reply:
Of course. That was a test. You passed :p

Reply:
I'll also remove that misleading comment a few lines later